test/e2e/apps/daemon_set.go

```diff
@@ -174,6 +174,33 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 		framework.ExpectNoError(err, "error waiting for daemon pod to revive")
 	})
 
+	/*
+	  Release: v1.10
+	  Testname: DaemonSet-ListAndDeleteCollection
+	  Description: A conformant Kubernetes distribution MUST list all DaemonSets and delete a collection of
+	  DaemonSets with a label selector.
+	*/
+	framework.ConformanceIt("should list all daemon and delete a collection of daemons with a label selector", func() {
```

Comment:
We cannot add conformance test directly without checking stability of conformance candidate tests.
Please use `ginkgo.It` instead of `framework.ConformanceIt` as a first step.